### DISCUSSION TITLE
Cannot log in to ozu!lazer (Incorrect sign in)

### DISCUSSION THREAD

**Discussion by HAHALOSAH** [ppy/osu#21348] [Q&A] [answered]
### Type

Game behaviour

### Bug description

When I try to log in to ozu!lazer with my login details that I know are correct, it fails to log in and says "Incorrect sign in"

### Screenshots or videos

![image](https://user-images.githubusercontent.com/67280050/202869051-cc818404-6390-4822-b2ed-2e6eba526583.png)

### Version

local debug

### Logs

[network] 2022-11-19 19:52:20 [verbose]: Request to https://dev.ppy.sh/oauth/token failed with System.Net.WebException: BadRequest.
[network] 2022-11-19 19:52:20 [verbose]: Response was: {"error":"invalid_grant","error_description":"The provided authorization grant (e.g., authorization code, resource owner credentials) or refresh token is invalid, expired, revoked, does not match the redirection URI used in the authorization request, or was issued to another client.","hint":"Incorrect sign in","message":"The provided authorization grant (e.g., authorization code, resource owner credentials) or refresh token is invalid, expired, revoked, does not match the redirection URI used in the authorization request, or was issued to another client."}
[network] 2022-11-19 19:52:20 [verbose]: Login failed for username HAHALOSAH (Incorrect sign in)!

**Comment by bdach** [comment, marked as answer]
The log shows that you're trying to run the game in debug mode, and you're connecting to `dev.ppy.sh` to log in rather than the real website (at `osu.ppy.sh`). The dev environment does not share logins with production.

If you wish to log in to an online account in dev mode, you're going to have to make one. Unless you're saying that you've made one already and that is the one that doesn't work?

**Comment by HAHALOSAH** [comment]
Does making another account go against the rule of multiaccounting?

**Reply by ghost** [reply]
if it's made on the `dev.ppy.sh` server, no.

**Reply by HAHALOSAH** [reply]
Just wondering, how are people able to get ranked plays on the development server?

**Reply by Theighlin** [reply]
You should be able to submit plays and see leaderboards on the development server.
If by ranked plays you mean ranking up the global leaderboard or awarding user statistics of the play, then you're gonna have to play on the official release of the game, with your only official osu ccount.
Of course, ranking up the global leaderboard is not yet possible in lazer.

**Reply by HAHALOSAH** [reply]
Then how were they able to rank up

**Reply by Theighlin** [reply]
By playing osu stable, downloadable on the official osu website https://osu.ppy.sh/home/download
